Dockerfile

```diff
@@ -92,17 +92,17 @@ RUN localedef -i en_US -f UTF-8 en_US.UTF-8
 # specify sphinx version as 1.5.6 and remove -U option for [pip install -U
 # sphinx-rtd-theme] since -U option will cause sphinx being updated to newest
 # version(1.7.1 for now), which causes building documentation failed.
```

Comment:
please add a comment why `py-cpuinfo==4.0.0` in next PR.

Reply:
please check the issue and the reference tells

Reply:
It is more easier for developers to find comments in the codes than use `git blame` to find in the PRs or related issues.

Reply:
no need add all comments in our source code, this is not some kind of note or our bug for developers , we only blame when we found errors. Just like other packages with specific version we did not need to add comment one by one.
And the commit description already tells the simple reason.